index.yaml

```diff
@@ -34550,46 +34578,6 @@ entries:
     urls:
     - assets/remotedialer-proxy/remotedialer-proxy-106.0.1+up0.5.0.tgz
     version: 106.0.1+up0.5.0
-  - annotations:
```

Comment:
I have no idea why was this removed and if it is fine or not

Reply:
The change should be fine